packages/blocks/src/_common/adapters/markdown.ts

```diff
@@ -445,6 +448,134 @@ export class MarkdownAdapter extends BaseAdapter<Markdown> {
               'children'
             )
             .closeNode();
+          break;
+        }
+        case 'affine:database': {
+          const rows: TableRow[] = [];
+          const columns = o.node.props.columns as Array<Column>;
+          const children = o.node.children;
+          const cells = o.node.props.cells as SerializedCells;
```

Comment:
We shouldn't use the `SerializedCells` type because `content-parser` a legacy module and we'll remove it soon.